facets_overview/common/utils.ts

```diff
@@ -472,6 +473,17 @@ export function cleanProto(datasets: DatasetFeatureStatisticsList):
   // deprecated field into the non-deprecated field.
   datasets.getDatasetsList().forEach(dataset => {
     dataset.getFeaturesList().forEach(feature => {
+      // If the feature's path step list is set, then use this path to create
+      // the feature's name, separated by forward slashes.
+      const path = feature.getPath();
+      if (path != null) {
+        const steps = path.getStepList();
+        if (steps != null) {
+          feature.setName(steps.reduce(
+            (accumulator, currentValue) => accumulator + '/' + currentValue));
```

Comment:
If `steps` is a regular JS Array, is this the same as `steps.join('/')`?

Reply:
done